src/pipeline.py

```diff
@@ -266,9 +267,61 @@ def __init__(self, **kwargs):
         )
 
 
+class ONNX_Pipeline(Pipeline):
+    def __init__(
```

Comment:
The vast majority of this function is identical to the base class. Seems like you just need to override methods like `_get_config, `_create_model`, etc. and add some assertions like in `DS_Pipeline`